src/adapter.py

```diff
@@ -34,21 +34,21 @@ def to_json(self, **table_kwargs):
             In this Creator service, it converts data from the Database to the initial JSON schema following the TNM.
         """
         dataModel = DataModel
-        nodesJSON = table_kwargs["nodes"]
-        dataModel.errorList = table_kwargs["errors"]
-        dataModel.metadata = table_kwargs["metadata"]
-        dataModel.vehicle = table_kwargs["vehicle"]
+        #nodesJSON = table_kwargs["nodes"]
+        # dataModel.errorList = table_kwargs["errors"]
+        # dataModel.metadata = table_kwargs["metadata"]
+        # dataModel.vehicle = table_kwargs["vehicle"]
 
         #Add nodes to dataModel
         for row in table_kwargs["nodes"]:
             node=makeNode(row)
-            dataModel.nodes.insert(node.node_id, node)#add node at its node_id
+            dataModel["nodes"].insert(node["node_id"], node)#add node at its node_id
```

Comment:
Som udgangspunkt er jeg nok lettere uenig i at sætte kode kommentarer for hvad koden gør.

Tror grunden til vi gør det nu, er fordi python jo er et nyt sprog, så vi kan godt lade det stå, hvis det er.